manifests/v/Vonage/VonageBusiness/2.11.0/Vonage.VonageBusiness.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Vonage.VonageBusiness
+PackageVersion: 2.11.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://storage.googleapis.com/ele-it-jumpcloud/VonageBusinessSetup.exe

```

```suggestion
  InstallerUrl: https://vbc-downloads.vonage.com/win/VonageBusinessSetup.exe
```
from https://www.vonage.com/downloads/

InstallerSha256 is the same (ed7c59c79dfaf31f3258d127325d15e9db6c8c9f795299c41e7d56f2da768b53)